org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/AcquisitionDevice.java

```diff
@@ -405,22 +417,30 @@ private void processException(Exception ne) throws ScanningException {
 
 	}
 	
-	private ScanInformation getScanInformation(int size) {
+	private ScanInformation getScanInformation(int size) throws ScanningException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "getScanInformation" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)